api/submission/fetch_submission_history_test.go

```diff
@@ -0,0 +1,122 @@
+package submission
+
+import (
```

Comment:
Import groupings are off.
It should be:
 - stdlib
 - third party
 - this project

Reply:
Fixed and noted!